src/generate.js

```diff
@@ -1,241 +1,64 @@
 const fs = require("fs");
 const path = require("path");
 const glob = require("glob");
-const assert = require("assert");
-const { URLSearchParams } = require("url");
-
-const fetch = require("node-fetch");
-const Validator = require("jsonschema").Validator;
-const Ajv = require("ajv");
-const addFormats = require("ajv-formats");
-const { schema } = require("@uniswap/token-lists");
-const { ethers } = require("ethers");
-const { sleep } = require("@eth-optimism/core-utils");
 
 const { version } = require("../package.json");
-const { TOKEN_DATA_SCHEMA } = require("./schemas");
 const { NETWORK_DATA } = require("./chains");
-const errors = require("./errors");
-
-const tokenABI = require("./abi/token.json");
-const factoryABI = require("./abi/factory.json");
 
 /**
  * Base URL where static assets are hosted.
  */
 const BASE_URL = "https://ethereum-optimism.github.io";
 
 /**
- * Call this to make CI require manual review.
+ * Generates a token list from the data in the data folder.
```

Comment:
Cuts generation down to literally just generating -- you can validate via `validate` if you want that